www/src/docs/elements/Banner.mdx

```diff
@@ -6,6 +6,12 @@ wip: true
 <Tabs>
   <Tab name="Code">
 
+### Usage
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis
+nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+
```

Comment:
In terms of MDX, note that i'm interpolating markdown inside of JSX (`<Tab>`); however -- the markdown has to be left justified for it to behave properly. 

Being able to mix and match like this is so totally 👌 -- JSX as more sane version of short-codes.